src/rules/typeLiteralDelimiterRule.ts

```diff
@@ -20,62 +20,95 @@ import * as ts from "typescript";
 
 import * as Lint from "../index";
 
+const OPTION_TYPE_ALIAS_DELIMITER: keyof Options = "type-alias-delimiter";
+
 export class Rule extends Lint.Rules.AbstractRule {
     /* tslint:disable:object-literal-sort-keys */
     public static metadata: Lint.IRuleMetadata = {
         ruleName: "type-literal-delimiter",
         description: Lint.Utils.dedent`
-            Checks that type literal members are separated by semicolons.
-            Enforces a trailing semicolon for multiline type literals.`,
-        optionsDescription: "Not configurable.",
-        options: null,
-        optionExamples: [true],
+            Checks that type literal members are separated by commas.
+            Enforces a trailing delimiter for multi-line type literals.`,
+        optionsDescription: Lint.Utils.dedent`
+            An options object may be passed with the following keys:
+
+            * '${OPTION_TYPE_ALIAS_DELIMITER}': This may be set to one of:
+              - \`","\`: This is the default. Type aliases are treated the same as other type literals.
```

Comment:
I think it would be easier to read the tslint.json configuration if these options were called `"semicolon" | "comma"`, what do you think?